.storybook/preview-head.html

```diff
@@ -1,27 +0,0 @@
-<link rel="preconnect" href="https://fonts.googleapis.com" />
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-<link
-  href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap"
-  rel="stylesheet"
-/>
-
-<style>
-  * {
-    --gray-pre-bg: #f8f8f8;
-    --font-family-monospace: Inconsolata, monospace;
-    --font-size-code: 0.875rem;
-  }
-
-  .docblock-source {
-    background: var(--gray-pre-bg) !important;
-  }
-
-  .docblock-source code {
-    font-family: var(--font-family-monospace) !important;
-    font-size: var(--font-size-code) !important;
-  }
-  .docblock-source code * {
-    font-family: var(--font-family-monospace) !important;
-    font-size: var(--font-size-code) !important;
-  }
-</style>
```

Comment:
To make the code blocks work for dark mode it's easier to remove and just accept the default styles

### Before

<img width="1104" alt="Screenshot 2023-11-11 at 10 17 39 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/80197f84-a7de-4893-b074-a0bbc77cd66d">

### After

<img width="1088" alt="Screenshot 2023-11-11 at 10 17 19 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d2ba93f2-b30a-4e3d-8591-4e0c4ce0b7b6">
<img width="1052" alt="Screenshot 2023-11-11 at 10 18 17 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/81a7f404-f617-478b-97cc-1927ad91326b">